Content.Shared/Kitchen/SharedKitchenSpikeSystem.cs

```diff
@@ -0,0 +1,31 @@
+using Content.Shared.DragDrop;
+using Content.Shared.Kitchen.Components;
+using Content.Shared.Nutrition.Components;
+
+namespace Content.Shared.Kitchen;
+
+public abstract class SharedKitchenSpikeSystem : EntitySystem
+{
+    public override void Initialize()
+    {
+        base.Initialize();
+        SubscribeLocalEvent<SharedKitchenSpikeComponent, CanDropOnEvent>(OnCanDrop);
+    }
+
+    private void OnCanDrop(EntityUid uid, SharedKitchenSpikeComponent component, ref CanDropOnEvent args)
```

Comment:
Its not super important, but the CanDropOn & CanDrop are names are still ambiguous to me, and the subject of the "question" could be either the dragged entity or target. Maybe make it CanReceiveDrag or some other better name to make it clearer which event is which?